File: Source/Rio.Models/DataTransferObjects/WaterTransfer/WaterTransferRegistrationDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Rio.Models.DataTransferObjects.WaterTransfer
{
    public class WaterTransferRegistrationDto
    {
        [Required]
        public int WaterTransferTypeID { get; set; }
        [Required]
        public int UserID { get; set; }
        public int? WaterTransferRegistrationStatusID { get; set; }
        public DateTime? StatusDate { get; set; }
        public List<WaterTransferRegistrationParcelDto> WaterTransferRegistrationParcels { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/water-transfer-registration-dto.ts
import { WaterTransferRegistrationParcelDto } from './water-transfer-registration-parcel-dto';

export class WaterTransferRegistrationDto {
    WaterTransferTypeID: number;
    UserID: number;
    WaterTransferRegistrationStatusID: number;
    StatusDate: Date;
    WaterTransferRegistrationParcels: Array<WaterTransferRegistrationParcelDto>;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

